src/AWS.jl

```diff
@@ -1,5 +1,6 @@
 module AWS
 
+using Compat
```

Comment:
Just include the module as that is all you need here
```suggestion
using Compat: Compat
```